template/base.html.jinja

```diff
@@ -194,35 +207,37 @@ Answer (including related code snippets if available and markdown links to relev
         `.trim(),
       },
       references: {
+        display: 'end',
         getHref: (reference) => {
-        const { file } = reference;
-        if (file && file.path) {
-          // /\/content\/[^_][^/]*\.md/.test(file.path)
-          if (file.path.includes('content/') && !file.path.includes('content/_') && file.path.endsWith('.md')) {
-            // Remove the extension if it's a regular content page that lives on xrpl.org
-            // return file.path.replace(/\.[^.]+$/, '');
-            // Change extension to html and remove content/ from the URL
-            if (file.path.endsWith('ja.md')) {
-              // Japanese lives at /ja/<filename>.html
-              return `ja/${file.path.replace(/content\//, '').replace(/\.ja\.md$/, '.html')}`
+          const { file } = reference;
+          if (file && file.path) {
+            // /\/content\/[^_][^/]*\.md/.test(file.path)
+            if (file.path.includes('content/') && !file.path.includes('content/_') && file.path.endsWith('.md')) {
+              // Remove the extension if it's a regular content page that lives on xrpl.org
+              // return file.path.replace(/\.[^.]+$/, '');
+              // Change extension to html and remove content/ from the URL
+              if (file.path.endsWith('ja.md')) {
+                // Japanese lives at /ja/<filename>.html
+                return `ja/${file.path.replace(/content\//, '').replace(/\.ja\.md$/, '.html')}`
+              }
+              return file.path.replace(/content\//, '').replace(/\.md$/, '.html')
+            } else {
+              // Otherwise link to a file on the github repo such as .js or README.md
+              return `https://github.com/XRPLF/xrpl-dev-portal/blob/master/${file.path}`;
```

Comment:
This section's indentation is off which makes it much harder to review because it creates unnecessary diffs. It may seem like a small thing, but it can also mess up the Git blame history and cause future developers to come ask you to explain this code instead of the original writers as this will show up as the last change on this line. I'd recommend skimming through this PR and fixing the indentation of any lines which are actually the same on both sides of the diff.

(If you setup a prettier config I think you can automatically have this formatting stuff fix itself)

Reply:
thanks for fixing the indentation - I went back and checked things - I commented out the algolia part since it's not used anymore - but might be useful in the future so I didn't delete it

Reply:
You can probably just delete it - if someone needs it they can refer to the git history :)